Instruct LLM to omit the addition of triple backticks

Issues like this are commonplace:
```python3

    @mark.parametrize("provider", ["ollama", "openai"])
    def test_read_instructions_from_file(
        provider: str, file_to_edit: Path, instructions_file: Path
    ) -> None:
        assert_command_success(
            f"{file_to_edit}",
            f"--provider={provider}",
            f"--file={instructions_file}",
            f"--output={file_to_edit}",
        )
>       assert "The sum is 14" in assert_python_script_runs(file_to_edit)
                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
...
tests/test_edit.py:75:
...
    def assert_python_script_runs(python_script: Path) -> str:
        # some tests will edit a python script with LLMs and place the edited copy elsewhere
        # run the edited copy to see if it works after modification by an LLM
        process = run(["python3", python_script], stdout=PIPE, stderr=PIPE, text=True)
>       assert process.returncode == 0, process.stderr
               ^^^^^^^^^^^^^^^^^^^^^^^
E       AssertionError:   File "/tmp/funcgraft_test_file.py", line 1
E           ```python
E           ^
E       SyntaxError: invalid syntax

tests/utils.py:26: AssertionError
```
This could be because during the prompt generation step, the program augments the prompt with triple backticks to provide some clues as to the which language we are working with:
https://github.com/dsw7/FuncGraft/blob/4a46de60000de3c9ef5f1762f52ae9821923a702/src/prompt/generate_prompt.cpp#L52-L76

The problem is that this coaxes the LLM to return the equivalent backtick wrapped code representation which in turn triggers the syntax error when the unit tests actually try to run the edited code.

--- a/src/adapters/components.cpp
+++ b/src/adapters/components.cpp
@@ -11,6 +11,9 @@ std::string get_system_prompt()
 You are a helpful assistant that specializes in programming.
 The user will provide some code and instructions on what to do with the the code.
 
+IMPORTANT: Do not wrap your response in backticks (```). Output the code
+directly without markdown code fences.
+
 If the query makes sense and is related to programming, then:
   Set `was_refused` to false.
   Set `code` to your updated code.

--- a/tests/test_slow.py
+++ b/tests/test_slow.py
@@ -57,7 +57,7 @@ def test_omit_code_fences() -> None:
     errors = []
 
     print("\n")
-    for n in range(1, 11):
+    for n in range(1, 51):
         try:
             edit_test_file(python_file, n)
         except AssertionError as e:

